simulations/vip-486/bscmainnet.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { forking, testVip } from "src/vip-framework/index";
+
+import { expectEvents } from "../../src/utils";
+import vip486, {
+  FAIRYPROOF_AMOUNT,
+  FAIRYPROOF_RECEIVER,
+  PESSIMISTIC_AMOUNT,
+  PESSIMISTIC_RECEIVER,
+  USDT,
+} from "../../vips/vip-486/bscmainnet";
+import ERC20_ABI from "./abi/ERC20.json";
+import VTREASURY_ABI from "./abi/VTreasury.json";
+
+const { VTREASURY } = NETWORK_ADDRESSES.bscmainnet;
+
+forking(48602303, async () => {
+  const usdt = new ethers.Contract(USDT, ERC20_ABI, ethers.provider);
+  let prevUSDTBalanceOfTreasury: BigNumber;
+  let prevUSDTBalanceOfPessimistic: BigNumber;
+  let prevUSDTBalanceOfFairyproof: BigNumber;
+
+  before(async () => {
+    prevUSDTBalanceOfTreasury = await usdt.balanceOf(VTREASURY);
+    prevUSDTBalanceOfPessimistic = await usdt.balanceOf(PESSIMISTIC_RECEIVER);
+    prevUSDTBalanceOfFairyproof = await usdt.balanceOf(FAIRYPROOF_RECEIVER);
+  });
+
+  testVip("VIP-486", await vip486(), {
+    callbackAfterExecution: async txResponse => {
+      await expectEvents(txResponse, [VTREASURY_ABI], ["WithdrawTreasuryBEP20"], [3]);
+    },
+  });
+
```

Comment:
Could we add any checks on the deBridge contracts after the VIP execution? Do they emit an event confirming the order? Is any balance increased on their side?